hugegraph-dist/src/assembly/travis/install-cassandra.sh

```diff
@@ -19,7 +19,7 @@ set -ev
 
 TRAVIS_DIR=`dirname $0`
 CASS_DOWNLOAD_ADDRESS="http://archive.apache.org/dist/cassandra"
-CASS_VERSION="3.10"
+CASS_VERSION="4.0.10"
```

Comment:
After this PR merged, maybe we could remove useless bundled `Java8` env in ci (HBase also need update the version)
<img width="1021" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/17706099/2cee7b32-9495-44c2-b949-71b2cfd028b6">


Hbase Refer: (so we need upgrade to HBase 2.3+ in ci)  
![img_v2_5fbd54e3-6118-4d32-bfd7-6bd1dbf0b77g](https://github.com/apache/incubator-hugegraph/assets/17706099/18984a31-12cc-43a5-ac87-167d3587139a)